examples/data-objects/smde/src/index.ts

```diff
@@ -3,18 +3,48 @@
  * Licensed under the MIT License.
  */
 
+import { mountableViewRequestHandler } from "@fluidframework/aqueduct";
 import { IContainerContext } from "@fluidframework/container-definitions";
 import { ContainerRuntime } from "@fluidframework/container-runtime";
-import { IFluidDataStoreFactory } from "@fluidframework/runtime-definitions";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
 import { buildRuntimeRequestHandler } from "@fluidframework/request-handler";
-import { defaultRouteRequestHandler } from "@fluidframework/aqueduct";
-import { RuntimeFactoryHelper } from "@fluidframework/runtime-utils";
-import { SmdeFactory } from "./smde";
+import { IFluidDataStoreFactory } from "@fluidframework/runtime-definitions";
+import {
+	requestFluidObject,
+	RequestParser,
+	RuntimeFactoryHelper,
+} from "@fluidframework/runtime-utils";
+import { MountableView } from "@fluidframework/view-adapters";
+
+import React from "react";
+
+import { SmdeDataObject, SmdeFactory } from "./smde";
+import { SmdeReactView } from "./smdeView";
 
 const defaultComponentId = "default";
 
 const smde = new SmdeFactory();
 
+const viewRequestHandler = async (request: RequestParser, runtime: IContainerRuntime) => {
```

Comment:
I'm not familiar with our view pattern. Might be nice to add some brief docs to this for folks who are unfamiliar. Also, probably worth specifying the return type.

Reply:
I'm going to leave this one - container-views is the better/dedicated example of what this is doing for the average customer, but I'd personally even recommend against that.  Preferred path forward would be to get rid of the webpack-fluid-loader and then this pattern goes away.